docs/parsepdf-architecture.html

```
@@ -0,0 +1,361 @@
+<title>ParsePDF — Architecture Map</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)